setuper/default_healthFacility_form.py

```diff
@@ -0,0 +1,126 @@
+import uuid
+from datetime import datetime
+from submissions import submission2xml
+from fake import fake_person
+import random
+
+
+def setup_health_facility_level_default_form(account_name, iaso_client):
+    print("-- Setting up a default form for Health Facility level")
+    project_id = iaso_client.get("/api/projects/")["projects"][0]["id"]
+    org_unit_types = iaso_client.get("/api/v2/orgunittypes/")["orgUnitTypes"]
+    hf_out = [out for out in org_unit_types if out["name"] == "Formation sanitaire"][0]
+
+    org_unit_type_id = [out["id"] for out in org_unit_types if out["name"] == "Formation sanitaire"]
+
+    default_fosa_level_form_name = "Data for Health facility/Données Formation sanitaire"
+
+    sample_data = {
+        "id": None,
+        "name": default_fosa_level_form_name,
```

Comment:
```suggestion
        "name": "Data for Health facility/Données Formation sanitaire",
```

The variable `default_fosa_level_form_name` is superflous since it's only used 1 time.

You can hardcode the string in the dict if you want. It also avoids wondering what `fosa` means in `default_fosa_level_form_name`.